source/blog/2018-07-06-the-ember-times-issue-54.md

```diff
@@ -21,8 +21,16 @@ Check out what's been going in Emberland 🐹 these days:
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [Organising the Organisers](https://emberjs.com/blog/2018/06/30/organizing-our-contributors.html)
+The Ember Team has published a new blog post outlining some structural changes to the core teams. It contains two changes: A new steering committee and a renaming of some of the existing teams.
```

Comment:
As you already wrote below, I think it would be cool to also capitalize the proper nouns that are used here:

`[...] changes to the Core teams [...]` 

and 

`[...] changes: A new Steering Committee and [...]`

Reply:
As a Dane, I will never learn this 🙈 Good catch